lib/sles4sap_publiccloud.pm

```diff
@@ -667,4 +673,96 @@ sub create_hana_vars_section {
     return (\%hana_vars);
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns 1 if hana database is online otherwise 0 offline.
+
+=back
+=cut
+
+sub get_hana_database_info {
+    my ($self, $password_db, $instance_id) = @_;
+    my $hana_database_online = '"HDB","HDB-$instance_id","YES","ACTIVE","","","DEFAULT","?"';
```

Comment:
Is `$instance_id` expected to be expanded? At the moment seems to me that it is not

```
my $instance_id = '123456';
my $hana_database_online = '"HDB","HDB-$instance_id","YES","ACTIVE","","","DEFAULT","?"';
print "hana_database_online:$hana_database_online \n";
```
result in 
```
hana_database_online:"HDB","HDB-$instance_id","YES","ACTIVE","","","DEFAULT","?" 
```
is it the expected behavior?

Reply:
good eye, no need this variable thx